src/os/shared.rs

```diff
@@ -82,6 +82,15 @@ fn sigwinch() -> (Box<OnSigWinch>, Box<SigCleanup>) {
     (Box::new(on_winch), Box::new(cleanup))
 }
 
+#[cfg(any(target_os = "windows"))]
+fn sigwinch() -> (Box<OnSigWinch>, Box<SigCleanup>) {
```

Comment:
Is there a better way to return sigwinch stub on Windows?

Reply:
Windows does not provide SIGWINCH, and its not that easy to get the console resize event.  https://stackoverflow.com/questions/10856926/sigwinch-equivalent-on-windows

I've just checked and yes, 

>  Something like: change size, see a broken interface which gets fixed after ~1 second?

that is how it behaves, however I noticed that only after you described that.
I think it is a cosmetical issue, and we could leave that for later.